charts/atlantis/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: v0.21.0
+appVersion: v0.22.0
```

Comment:
The appversion is not the latest version here

Reply:
appVersion in chart v4.9.2 was not reflecting the app version deployed by default, because the values.yaml was setting the tag to v0.22.0, I just wanted to realign these two values to avoid any confusion. I'd prefer not to update to a new version here, as I haven't tested it.